css/decima.css

```
@@ -0,0 +1,77 @@
+.page-content-wrapper{
+  background-color: white;
+  border: none;
+}
+/*Logo de la empresa*/
+.logo{
+    align-content: center;
+  }
+.content-viewport{
+  background: white;
+  border: none;
+}
+.grid{
+  border: none;
+}
+.grid-body{
+  background-color: white;
+  border: none;
+}
+/*Titulo*/
+.titulo{
+  font-size: 30px;
+  text-align: center;
+  padding: 13px;
+}
+.tab-container .tab-content {
+  background: #FDFDFD;
+}
+/*Lista principal*/
+.lista-principal{
+  font-weight: 900;
+  font-size: 14px;
+}
+.lista-principal2{
+  font-weight: 900;
+  font-size: 14px;
+}
+.lista-principal3{
+  font-weight: 900;
+  font-size: 14px;
+}
+/*Lista secundaria*/
+a.lista-secundaria{
+color: #ddcb13;
+font-weight: 900;
+}
+a.lista-secundaria2{
```

You are repeating code,CSS allows you to group multiple selectors that share the same declaration. This optimization technique allows you to apply the same style to multiple elements to save space.
```suggestion
a.lista-secundaria2, a.lista-secundaria3, a.lista-secundaria3 {
 color: #ddcb13;
 font-weight: 900;
}

```

---

[This are the Docs for CSS sintax](https://www.w3schools.com/css/css_syntax.asp)